Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/InputAction.cs

```diff
@@ -22,6 +22,7 @@ public InputAction(uint id, string description)
         {
             this.id = id;
             this.description = description;
+            this.axis = AxisType.None;
```

Comment:
Shouldn't this be getting passed into the constrictor?

Reply:
Axis is Optional, which is why there was an overload for the AxisType.  I left it off orrigially as it broke the Inspector, just added the overload and it's now fine?  Oh well.